### DISCUSSION TITLE
After uploading proof documents, the rear camera still cannot be invoked

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65360] [GitHub Education]
I tried it, but uploading the proof file still fails to call the rear camera
![IMG_20230829_232141](https://github.com/community/community/assets/142904862/708a0bf4-6622-4bb3-ac7c-d09c24d6f7a1)

**Comment by ghost** [comment]
Officials still haven't solved the problem

**Comment by arafajs** [comment]
1. Check that the camera app is up to date.
2. Restart your phone.
3. Clear the cache and data for the camera app.
4. Factory reset your phone.

**Reply by ghost** [reply]
The front camera of the iPhone is good, which means that it is not the problem of the mobile phone, nor the problem of the camera; Android can call the rear camera, but the picture is not clear
Obviously, the official needs to fix this problem, the official should have received feedback from users, however, the official has no idea of ​​fixing this problem

**Comment by h1ac** [comment]
I used the opera browser and It let me choose the back camera! I got accepted :)
